apps/emqx/etc/emqx.conf

```diff
@@ -1645,7 +1645,7 @@ persistent_session_store {
     ## @doc persistent_session_store.enabled
     ## ValueType: Boolean
     ## Default: false
-    enabled = false
+    enabled = true
```

Comment:
Forgotten change?

Reply:
Easy for testing. If no problems, I will reset it to true.